src/components/SignInModal.tsx

```diff
@@ -26,13 +37,64 @@ export const SignInModal: React.FC<SignInModalProps> = ({
         onClose();
       })
       .catch(e => {
+        if (e.code == 'auth/account-exist-with-different-credential') {
```

Comment:
is this a typo?
```suggestion
        if (e.code === 'auth/account-exists-with-different-credential') {
```
maybe make this a variable to avoid typos :p